docroot/modules/custom/sitenow_signage/sitenow_signage.module

```diff
@@ -5,17 +5,114 @@
  * Module code for SiteNow Signage.
  */
 
-use Drupal\Component\Utility\Html;
+use Drupal\Core\Form\FormStateInterface;
 use Drupal\Core\Render\Element;
+use Drupal\layout_builder\Plugin\SectionStorage\OverridesSectionStorage;
 use Drupal\node\NodeInterface;
 
+/**
+ * Implements hook_module_implements_alter().
+ */
+function sitenow_signage_module_implements_alter(&$implementations, $hook) {
+  if ($hook == 'form_alter') {
```

Comment:
Can we add a comment here explaining what is happening and why? I'm not understanding it just by reading the code.

Reply:
I guess I'm surprised that `sitenow_signage` hooks don't fire after `layout_builder_custom`, since it comes later alphabetically.